packages/actions/src/helpers/s3.ts

```diff
@@ -0,0 +1,320 @@
+import { Functions, httpsCallable, HttpsCallable } from "firebase/functions"
```

Comment:
Bonus: rename this file `storage.ts`. I would like to keep things separated from the tech behind. In this case, AWS S3 will be our only one storage. I would like to keep also methods name as abstract as possible (just think about the storage concept). Thanks <3